literature.bib

```diff
@@ -900,6 +955,18 @@ @article{SHG+:JEA23
   address = NY
 }
 
+@inproceedings{ASP+:ICDM23,
+	author={Abolfazli, Amir and Spiegelberg, Jakob and Palmer, Gregory and Anand, Avishek},
+	title={{A Deep Reinforcement Learning Approach to Configuration Sampling Problem}}, 
+	booktitle={ICDM},
+	pages={1-10},
+	publisher = IEEE,
+	keywords={Deep learning;Costs;Software algorithms;Reinforcement learning;Sampling methods;Software systems;Software product lines;reinforcement learning;configuration sampling;software testing},
```

Comment:
typically not separated with semicolons here.

Reply:
We use commas.